sending-email.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Sending email
+layout: recommendation
+expires: 2017-11-01
+---
+
+## Using GOV.UK Notify
```

Comment:
can we express the section headers in terms of the need, not the solution? ie "sending notifications to users" rather than "GOV.UK Notify"